docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md

```
@@ -0,0 +1,168 @@
+   export TRUST_HASH=$(curl -s "https://rpc.chiado.wardenprotocol.org/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
+   ```
+
+2. Check that all variables have been set correctly:
+
+   ```bash
+   echo $LATEST_HEIGHT $BLOCK_HEIGHT $TRUST_HASH
+   ```
+
+   The output should be similar to this:
+
+   ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)